interop/ksp/test-processor/src/main/kotlin/com/squareup/kotlinpoet/ksp/test/processor/exampleAnnotations.kt

```diff
@@ -79,3 +79,5 @@ annotation class AnotherAnnotation(val input: String)
 enum class AnnotationEnumValue {
   ONE, TWO, THREE
 }
+
+annotation class AnnotationWithVararg(vararg val args: String)
```

Comment:
Can we add one with mixed vararg/regular arg?

Reply:
Added to the existing annotation since, though I could split it out into 2 tests if you feel it's important